app/assets/stylesheets/modules/profile.scss

```diff
@@ -21,3 +21,7 @@
   border-width: 8px;
   border-radius: 8px;
 }
+#review_author {
```

Comment:
Avoid using id selectors<br>Selector `review_author` should be written in lowercase with hyphens